src/lib/Scenes/Home/index.js

```diff
@@ -0,0 +1,83 @@
+"use strict";
```

Comment:
I think this may have been committed by accident?

Reply:
Interesting! I just removed it - let's see if that fixes the CI.